src/api/environment.cc

```diff
@@ -362,6 +362,16 @@ Local<Context> NewContext(Isolate* isolate,
                           Local<ObjectTemplate> object_template) {
   auto context = Context::New(isolate, nullptr, object_template);
   if (context.IsEmpty()) return context;
+
+  if (!InitializeContext(context, object_template)) {
+    return Local<Context>();
+  }
+  return context;
+}
+
+bool InitializeContext(Local<Context> context,
+                       Local<ObjectTemplate> object_template) {
```

Comment:
Why does this take the `object_template`? I don't think it's used in initialization?

Reply:
Good question 😆 looks like I copy-pasted it without checking it it was actually used, removed 👍